### ISSUE TITLE
Toolbox bug? Can't select z-offset setting plugin

### ISSUE THREAD

**Issue by smartavionics** [Ultimaker/Cura#3769]
**Application Version**

master

**Platform**

Linux

**Printer**

**Steps to Reproduce**

Open toolbox, move mouse over various plugin entries

**Actual Results**

Moving the mouse over the z-offset plugin entry doesn't highlight it and the entry cannot be selected.

**Expected results**

You know...

**Additional Information**

Notice how the area under the mouse cursor is not highlighted blue.

![screenshot_2018-05-03_08-27-55](https://user-images.githubusercontent.com/585618/39564467-489e43c8-4eac-11e8-9954-fc87276ad66a.png)

**Comment by ChrisTerBeke** [comment]
Confirmed issue.

**Comment by ianpaschal** [comment]
Yes. Has something to do with how QT handles grid layouts. :/

It's on our "leftover bugs" list.

**Comment by ChrisTerBeke** [comment]
I saw that if you add some padding/spacing on the bottom, it becomes clickable again :P

**Comment by ianpaschal** [comment]
Yes that's what I thought was in there... there's some line where I check the remainder of the model length and if it's odd add some extra spacing but for some reason that stopped (ehhh?) working?

**Comment by ianpaschal** [comment]
Should be in `ToolboxDownloadsGrid.qml`

**Comment by diegopradogesto** [comment]
I made a fix in 2283bb530debdcbe6ac4c544f7920377c1cb01dc. Please try if it works.

**Comment by fieldOfView** [comment]
Seems to work, be sure to remove the comment above the fix.

**Comment by smartavionics** [comment]
Just tried that and it appears to be OK now, thanks. Closing...
